src/cider/nrepl.clj

```diff
@@ -22,6 +22,11 @@
    [orchard.namespace]
    [orchard.java]))
 
+(when (and (< (-> *clojure-version* :major long) 2)
```

Comment:
I'd probably move the required version to a constant (a map would be best I guess), so it's clearer what we're doing.